tests/test_regression.py

```diff
@@ -51,3 +54,27 @@ class Params:
 
         unknown_author = AuthorFactory(unknown=True)
         self.assertEqual("", unknown_author.fullname)
+
+
+class NameConflictTests(unittest.TestCase):
+    def test_name_conflict_issue(self):
+        """Regression test for `TypeError: _save() got multiple values for argument 'x'`
```

Comment:
Using session as the attribute name helps understand what the test is about. That’s also the text from issue #775.
```suggestion
        """Regression test for `TypeError: _save() got multiple values for argument 'session'`
```